tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -121,7 +121,7 @@ def _prepare_input(self):
         raise RuntimeError("Please provide inputs generation function")
 
 
-def get_params(ie_device=None, precision=None):
+def get_params(ie_device=['CPU'], precision=None):
```

Comment:
please remove. Lifehack: you can do only CPU testing by declaring env var
`export TEST_DEVICE=CPU`